commands/fun/coinflip.js

```diff
@@ -76,76 +60,61 @@ module.exports = {
 		const colFilter = (buttonInteraction) => {
 			buttonInteraction.deferUpdate();
 			return interaction.user.id === buttonInteraction.user.id;
-		}
+		};
 
 		const collector = message.createMessageComponentCollector({
-			componentType : "BUTTON",
-			max : 1,
-			filter : colFilter,
-			time: 1000 * 8,
+			componentType: "BUTTON",
+			max: 1,
+			filter: colFilter,
+			time: 1000 * 4,
 		});
 
 		// Collector End Event: Handles game logic and closure
 		collector.on("end", async (collected) => {
-
+			// User took longer than 8 seconds to respond
 			if (collected.size === 0) {
 				return await interaction.editReply({
 					content: `
 						> __**Game Cancelled**__
 						> Any bets have been refunded.
-						> Please click on the Heads or Tails buttons. The time limit is 8 seconds.`,
+						> Please click on the Heads or Tails buttons. The time limit is 4 seconds.`,
```

Comment:
Should this be four or eight seconds? Comment on L74 says eight.

Reply:
Good catch, I updated the text from 8 to 4 seconds.